pdf-review/components/PDFViewer.js

```diff
@@ -1,42 +1,74 @@
-import React, { useState } from "react"
+import React, { useEffect, useState } from "react"
 import { Document, Page } from "react-pdf"
 
 const PDFViewer = ({ url }) => {
-  const [rotation, updateRotation] = useState(270)
-  const [pageCount, updatePageCount] = useState(0)
-  const [zoom, updateZoom] = useState(2.1)
+  const [rotation, setRotation] = useState(270)
+  const [pageCount, setPageCount] = useState(0)
+  const [zoom, setZoom] = useState(2.1)
   const pages = []
 
   for (var i = 1; i <= pageCount; i++) {
     pages.push(i)
   }
 
+  function handleKeyUp(e) {
+    const { key, shiftKey } = e
+
+    if (!shiftKey) return
+
+    console.log({ key })
+
+    if (key === "ArrowRight") {
+      rotateClockwise(e)
+    } else if (key === "ArrowLeft") {
+      rotateCounterClockwise(e)
+    } else if (key === "+") {
+      zoomIn(e)
+    } else if (key === "_") {
+      zoomOut(e)
+    }
+  }
+
   function handleLoadSuccess(e) {
     const { numPages } = e._pdfInfo
-    updatePageCount(numPages)
+    setPageCount(numPages)
   }
 
   function rotateCounterClockwise(e) {
+    console.log('rcc', rotation, e)
     e.preventDefault()
-    if (rotation === 0) return updateRotation(270)
-    updateRotation(rotation - 90)
+    if (rotation === 0) return setRotation(270)
+    setRotation(rotation - 90)
   }
 
   function rotateClockwise(e) {
+    console.log("rc", rotation)
     e.preventDefault()
-    if (rotation === 270) return updateRotation(0)
-    updateRotation(rotation + 90)
+    console.log('yo!')
```

Comment:
I see this stuff get logged, I just don't see the result of the updated state (aka the PDF being rotated)